randovania/gui/multiplayer_session_window.py

```diff
@@ -373,7 +373,11 @@ async def on_meta_update(self, session: MultiplayerSessionEntry):
         self.allow_coop_check.setChecked(session.allow_coop)
         self.allow_coop_check.setEnabled(self.users_widget.is_admin() and not_genned_yet)
         self.allow_coop_check.setText(
-            "Enable Co-Op" + ("" if not_genned_yet else " (can only be changed before generation)")
+            "Co-Op Worlds" + ("" if not_genned_yet else " (can only be changed before generation)")
```

Comment:
You added the "can only be changed before generation" to the tooltip. I think that's very redundant now. As we now have a tooltip, I think it is sufficient to only have it there and it is much better than adding explanations dynamically to the label.